packages/next-server/lib/utils.js

```diff
@@ -63,7 +63,7 @@ export function formatWithValidation (url, options) {
   if (process.env.NODE_ENV === 'development') {
     if (url !== null && typeof url === 'object') {
       Object.keys(url).forEach((key) => {
-        if (!urlObjectKeys.includes(key)) {
+        if (urlObjectKeys.indexOf(key) === -1) {
```

Comment:
Interesting! It breaks on 8.0.3 on development when open with IE but not affecting production on IE

Reply:
@tomchentw the code was only executed in development, see a few lines above:

```
process.env.NODE_ENV === 'development'
```

😄